chef/roles/bmc-nat-client.rb

```diff
@@ -1,7 +1,7 @@
 name "bmc-nat-client"
 description "Sets up routes to access BMC addresses"
 run_list(
-         "recipe[bmc-nat::client]"
+         "recipe[ipmi::role_bmc_nat_client]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.